management_structure_and_procedures.tex

```diff
@@ -346,7 +346,11 @@ \subsubsection{Risks and risk management strategy}\label{sec:risks}
   a diverse range of needs and points of views; hence the design of
   the proposal and the governance of the project is naturally steered
   by demand; besides, because we provide a toolkit, users have the
-  flexibility to adapt the infrastructure to their needs.\\\hline
+  flexibility to adapt the infrastructure to their needs. In addition, the open source nature 
+  of the project facilitates and promotes the involvement of the wider community in terms of 
+  providing feedback and requesting additional features via platforms such as GitHuib and BitBucket 
```

Comment:
typo in GitHub

Bitbucket only capitalises the first B

Reply:
ooops sorry for all those typos! 🤦🏻‍♀️ and thanks for the extra pair of eyes

Reply:
No problem :-)